e2etest/bgp_test.go

```diff
@@ -72,30 +72,28 @@ type containerConfig struct {
 var _ = ginkgo.Describe("BGP", func() {
 	var cs clientset.Interface
 	var f *framework.Framework
-	containersConf := []containerConfig{
-		{
-			name: frrIBGP,
-			nc: frrconfig.NeighborConfig{
-				ASN:      IBGPAsn,
-				Password: "ibgp-test",
-			},
-			rc: frrconfig.RouterConfig{
-				ASN:      IBGPAsn,
-				BGPPort:  179,
-				Password: "ibgp-test",
-			},
+	ebgpContainerConfig := containerConfig{
+		name: frrIBGP,
+		nc: frrconfig.NeighborConfig{
+			ASN:      IBGPAsn,
+			Password: "ibgp-test",
 		},
-		{
-			name: frrEBGP,
-			nc: frrconfig.NeighborConfig{
-				ASN:      IBGPAsn,
-				Password: "ebgp-test",
-			},
-			rc: frrconfig.RouterConfig{
-				ASN:      EBGPAsn,
-				BGPPort:  180,
-				Password: "ebgp-test",
-			},
+		rc: frrconfig.RouterConfig{
+			ASN:      IBGPAsn,
+			BGPPort:  179,
+			Password: "ibgp-test",
+		},
+	}
+	ibgpContainerConfig := containerConfig{
+		name: frrEBGP,
+		nc: frrconfig.NeighborConfig{
+			ASN:      IBGPAsn,
+			Password: "ebgp-test",
+		},
+		rc: frrconfig.RouterConfig{
+			ASN:      EBGPAsn,
+			BGPPort:  180,
+			Password: "ebgp-test",
```

Comment:
if its ibgp container why its called frrEBGP and ASN should be the same for router and nbr ?

Reply:
Because I mismatched the names :-)